tests/jest.setup.js

```diff
@@ -9,6 +9,9 @@ import mockSafeAreaContext from "react-native-safe-area-context/jest/mock";
 
 import { mockCamera, mockSortDevices } from "./vision-camera/vision-camera";
 
+// eslint-disable-next-line max-len
```

Comment:
Please try and correct the problems the linter reports. Disabling the linter is only for situations where you really can't address the issue. In this case you can wrap this statement like this:

```javascript
jest.mock(
  "@react-native-async-storage/async-storage",
  () => require( "@react-native-async-storage/async-storage/jest/async-storage-mock" )
);
```